src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -48,12 +48,12 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EmailSender.class);
 
     @Getter
-    @Value("${email.sender}")
-    private String sender;
+    @Value("${email.sender:noreply@ericsson.com}")
+    private static final String sender = "noreply@ericsson.com";
```

Comment:
A "private static final" variable should always be named in uppercase. You could remove static or rename the variables.

Reply:
Can we have ericsson.com in here?

Reply:
Question is: what to have instead as a default sender ? 🤔  Any suggestions?

Reply:
In principle, I think this should be a configurable thing. To be strict, perhaps no default may be used?

Reply:
I think its possible/ok to leave the sender address field empty in a mail, -> defaulting to a empty string. Only issue can be that Mail lib maybe require a non-empty string on that parameter and throw an exception. You can maybe set it to no-reply@dummy.com or something.

Reply:
Yes I wrote this issue because I noticed the lib throws an exception if the sender is empty. That is why we want a default if this is NOT configured in application.properties. Admin users will have the option to configure the sender + subject (which they probably will) but in case they forget we have default values.

Reply:
Another approach is to write a Log message that Email is disabled when mail parameter is missing or failed to connect to SMTP server. And add Status info in Frontend Status button / info box that "Email is disabled" in backend.

Reply:
And leave the mail properties default empty and don't default anything. But when mail host property is specifued, some validation is required, e.g. senders mail address.